portfolio/src/main/webapp/script.js

```diff
@@ -30,10 +33,22 @@ function addRandomGreeting() {
     greetingContainer.innerHTML = greeting;
 }
 
-async function welcome() {
-    const response = await fetch('/data');
-    const quote = await response.text();
-    document.getElementById('welcome-container').innerHTML = quote;
+async function onLoad() {
+    await login();
+    await getComments();
+}  
+
+async function login() {
+    const response = await fetch('/login');
+    const welcome = await response.json();
+    if (welcome[0] === "") {
+        loginUrl = welcome[1];
+        document.getElementById('login-container').innerHTML = " <a href="+ welcome[1] +"> Login </a>";
+    } else {
+        userEmail = welcome[0];
+        loginUrl = welcome[1];
```

Comment:
Just as I commented before, use a JSON object might be better to understand what the fields are.